parse/parse.go

```diff
@@ -0,0 +1,319 @@
+package parse
```

Comment:
For this and the cli/common.go, it would probably be good to have a little documentation at the top listing the types of functions one should find in this piece of go code.